src/frontend/src/routes/(new-styling)/manage/(authenticated)/(access-and-recovery)/recovery/+page.svelte

```diff
@@ -6,10 +6,24 @@
   import { getMetadataString } from "$lib/utils/openID";
   import ActiveRecoveryPhrase from "./components/ActiveRecoveryPhrase.svelte";
   import { authenticatedStore } from "$lib/stores/authentication.store";
+  import Dialog from "$lib/components/ui/Dialog.svelte";
+  import { CreateRecoveryPhraseWizard } from "$lib/components/wizards/createRecoveryPhrase";
+  import { throwCanisterError } from "$lib/utils/utils";
+  import { invalidateAll } from "$app/navigation";
+  import UnverifiedRecoveryPhrase from "./components/UnverifiedRecoveryPhrase.svelte";
+  import { recoveryAuthnMethodData } from "$lib/utils/authnMethodData";
+  import type { AuthnMethodData } from "$lib/generated/internet_identity_types";
+  import { authnMethodEqual } from "$lib/utils/webAuthn";
 
   const { data }: PageProps = $props();
 
-  const recoveryPhrase = $derived(
+  let showRecoveryPhraseSetup = $state<"activate" | "reset" | "verify">();
+  let unverifiedRecoveryPhrase = $state<{
```

Comment:
Unverified means only in that session in the browser, right? On refresh it will disappear.